### DISCUSSION TITLE
Can Scene Viewer UI elements be customized?

### DISCUSSION THREAD

**Discussion by arberrexhepi** [google/model-viewer#1870] [Q&A]
Don't know if this is the right place for this BUT
Would love to know if it is or will be possible to have UI elements within Scene Viewer.
Basically I want to change the source of the animation while I am in AR mode, so my character can go from punching to let's say kicking. I would like to create a multiplayer AR game, so maybe even scene sharing via some type of verification would be great.

**Comment by cdata** [comment]
@arberrexhepi I answered a very similar question just a few days ago. Please see https://github.com/GoogleWebComponents/model-viewer/issues/745#issuecomment-531603192

Answer quoted here:

> It's definitely possible to customize `<model-viewer>`, but `<model-viewer>` (the HTML tag) is not the same thing as Scene Viewer (the native app that launches when you press the AR button on Android).
> 
> If you want to customize Scene Viewer, I'm afraid that is not possible at this time.
> 
> However, if you want to customize `<model-viewer>`, the sky is the limit. And, if you hit a limitation, we will probably do what we can to remove that limitation in a future version 👍

This answer is true for your case as well. Ideally, WebXR would exist so that we could enable your use case directly from the web. Hopefully in the next year this will be the case.

**Comment by 3dxxl** [comment]
I am also looking for implement Screen ui Elements in scene Viewer AR.

**Comment by jasimmj** [comment]
Any update on this?

**Reply by elalish** [reply]
This is unlikely to happen, and it's part of the reason we're focusing on WebXR over app-based AR. With WebXR you can put any HTML  elements you want as the  children of the MV element, and they'll show up in AR as well. And you can use the `ar-status` attribute to control their behavior with CSS. It's more or less impossible for a separate app to give that level of API power.
